test/clt-tests/sharding/replication/init.recb

```diff
@@ -0,0 +1,28 @@
+––– input –––
+set -b +m
+––– output –––
+––– input –––
+apt-get update -y > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+apt-get install -y git > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+git clone -b sharding/inserts -q https://github.com/manticoresoftware/manticoresearch-buddy.git /workdir; echo $?
```

Comment:
Why this file is the the same and duplicating same init.recb from sharding/base ?

Reply:
Discussed this, waiting for a fix and then I'll remove the init block.